python/pyspark/sql/connect/proto/pipelines_pb2.pyi

```diff
@@ -107,9 +111,10 @@ class PipelineCommand(google.protobuf.message.Message):
                 value: builtins.str = ...,
             ) -> None: ...
             def ClearField(
-                self, field_name: typing_extensions.Literal["key", b"key", "value", b"value"]
+                self, field_name: typing.Literal["key", b"key", "value", b"value"]
```

Comment:
I think this change is indicative of using a different proto gen library version than the one normally used in the build. Let me know if you need help installing the right version.